src/dashboard/apps/core/management/commands/syncdeliverypoints.py

```diff
@@ -51,9 +52,17 @@ def sync_delivery_points(self, siret: list[str] | None = None) -> None:
         for entity in entities:
             self._log_notice(f"⚙️ Processing SIRET: {entity.siret}...")
             try:
-                sync_delivery_points_from_qualicharge_api(entity)
+                delivery_points, consents = sync_delivery_points_from_qualicharge_api(
+                    entity
+                )
+                self._log_notice(f"- {len(delivery_points)} delivery point(s) created")
+
+                if consents:
+                    send_notification_for_awaiting_consents(entity)
```

Comment:
This should stand outside from this exception catching block: your intend is to catch specific errors from the `sync_delivery_points_from_qualicharge_api` helper.